angular/package.json

```diff
@@ -10,10 +10,13 @@
     "type": "git",
     "url": "https://github.com/TheIronYard--Orlando/TIY-Catalog"
   },
-  "author": "David Rogers <david@theironyard.com>",
+  "author": "David Rogers <david@theironyard.com>, Taylor Foster <tagfoster90@gmail.com>",
```

Comment:
Should use `"contributors"` instead re https://docs.npmjs.com/files/package.json#people-fields-author-contributors